qa/distros/all/ubuntu_20.04.yaml

```diff
@@ -1,2 +1,9 @@
 os_type: ubuntu
 os_version: "20.04"
+# the normal ubuntu 20.04 kernel (5.4.0-88-generic currently) have a bug that prevents the nvme_loop
+# from behaving.  I think it is this:
+#   https://lkml.org/lkml/2020/9/21/1456
+# (at least, that is the symptom: nvme nvme1: Connect command failed, error wo/DNR bit: 880)
+overrides:
+  kernel:
+    hwe: true
```

Comment:
This breaks the fs suite:

http://pulpito.front.sepia.ceph.com/yuriw-2022-01-19_19:27:28-fs-wip-yuri5-testing-2022-01-19-0916-pacific-distro-default-smithi/6627433/

and many others. Why wasn't this change part of `qa/distros/container-hosts/ubuntu_20.04.yaml ` as in the original commit?

https://github.com/ceph/ceph/pull/44505/commits/5c0f0698a5b8db75ae9bcdca311a68a1589ee0a5

(The cherry-pick should have noted the conflict!)

cc @vshankar

Reply:
> This breaks the fs suite:

sorry about that @batrick 

> http://pulpito.front.sepia.ceph.com/yuriw-2022-01-19_19:27:28-fs-wip-yuri5-testing-2022-01-19-0916-pacific-distro-default-smithi/6627433/
> 
> and many others. Why wasn't this change part of `qa/distros/container-hosts/ubuntu_20.04.yaml ` as in the original commit?

it seems it was different on pacific
```
$ ls -l qa/suites/rados/cephadm/smoke/distro/ubuntu_20.04.yaml
lrwxrwxrwx 1 guillaume guillaume 33 Jan 18 16:55 qa/suites/rados/cephadm/smoke/distro/ubuntu_20.04.yaml -> .qa/distros/all/ubuntu_20.04.yaml
```

that's why I changed it here in the pacific branch, not sure how but I must have got confused 😐